.github/workflows/notify-security-alerts.yml

```diff
@@ -0,0 +1,16 @@
+name: Send security alerts to slack
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+  workflow_dispatch:
+
+jobs:
+  send_security_alerts:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Send security alert notifications to slack
+        uses: kunalnagarco/action-cve@v1.8.0
+        with:
+          token: ${{ secrets.SECURITY_ALERTS_GITHUB_TOKEN }}
```

Comment:
I'm getting secops to provision us a token for this, will update the secret and the proper webhook url once it's all ready to go

Reply:
This is done